docs/intro/install.rst

```diff
@@ -34,6 +32,19 @@ To install using pip::
 Platform specific installation notes
 ====================================
 
+Anaconda
+--------
+
+.. note::
+
+  For Windows and OS X users, this is the recommended way to install Scrapy.
```

Comment:
I'd say on OS X Scrapy is easier to install with pip than on Linux because wheels work.